### ISSUE TITLE
Support for Non-Integer Percentile Calculations

### ISSUE THREAD

**Issue by pb-stripe** [apache/pinot#6301]
In evaluating Pinot for an [SLI/SLO](https://cloud.google.com/blog/products/gcp/sre-fundamentals-slis-slas-and-slos) analytics use case, I was curious if there's built-in support for percentiles above P99. Here's the resulting conversation in Slack:

```
Paul Baumgart Today at 10:42 AM
Q: is it possible to query non-integer percentiles? I'm interested in calculating P99.9, but looking at the supported aggregations doc, it looks like those functions only support up to P99.

Jackie  7 minutes ago
Yes, we can support that. Can you please file an issue on GitHub? This feature requires some change on the function name parsing, should be simple. Contributions are very welcome

Mayank  7 minutes ago
@Jackie with functions now taking multiple args, do we still need to rely on name parsing?

Jackie  6 minutes ago
Good point, we just need to parse double

Xiang Fu  6 minutes ago
right now the percentile only takes the integer
```

I will defer to folks experienced with the codebase to add more detail on how best to implement.

Thanks!

**Comment by amrishlal** [comment]
Support for decimal percentile has been added through https://github.com/apache/incubator-pinot/pull/6323.

**Comment by vijaya-lakshmi-venkatraman** [comment]
Hello,
Is this still available?

**Comment by Jackie-Jiang** [comment]
Close this per the fix in #6323
